src/Login/Login.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ChangeEventHandler, Component } from 'react';
+
+type State = {
+  email: '';
+  password: '';
+};
+
+export class Login extends Component {
+  public state: State = {
+    email: '',
+    password: '',
+  };
+
+  public render() {
+    return (
+      <div>
+        <input
+          placeholder='Email'
+          value={this.state.email}
+          onChange={this.onEmailChange}
+        />
+        <input
+          placeholder='Password'
```

Comment:
This should have a `type="password"`, else it isn't obfuscated and hidden from prying eyes.

Reply:
Got it!